src/main/java/com/internet/shop/controller/AddProductToStorageController.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.shop.controller;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddProductToStorageController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private ProductService productService = (ProductService) injector
+            .getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/product/addProduct.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
```

Comment:
What if we receive an empty input String?

Reply:
added validation of the form in jsp files